src/amo/components/MastHead.js

```diff
@@ -35,15 +30,12 @@ export class MastHeadBase extends React.Component {
         </div>
         {children}
         <header className="MastHead-header">
-          {isHomePage
-            ? <h1 ref={(ref) => { this.title = ref; }}
-                className="MastHead-title MastHead-homepage">
-              {headerTitle}
-            </h1>
-            : <Link ref={(ref) => { this.title = ref; }} to="/"
-                className="MastHead-title">
-              {headerTitle}
-            </Link>}
+          <h1>
```

Comment:
So there should be an h1 on the homepage for the title, and not elsewhere. The original behaviour would have done that but just need the isHomepage bool to be true for the homepage.